interface/forms/vitals/vitals.js

```diff
@@ -77,27 +83,65 @@
                     inputSave.value = value;
                 }
             } else {
-                console.error("Failed to get valid number for input with id ", node.id, " with value ", value);
+                console.error(
+                    "Failed to get valid number for input with id ",
+                    node.id,
+                    " with value ",
+                    value
+                );
             }
         } else {
             inputSave.value = "";
             inputConv.value = "";
         }
 
-        if (targetSaveUnit == "weight_input" || targetSaveUnit == "height_input") {
+        if (
+            targetSaveUnit == "weight_input" ||
+            targetSaveUnit == "height_input"
+        ) {
             calculateBMI();
         }
     }
 
     function initDOMEvents() {
-        let vitalsForm = document.getElementById('vitalsForm');
+        let vitalsForm = document.getElementById("vitalsForm");
         if (!vitalsForm) {
             console.error("Failed to find vitalsForm DOM Node");
             return;
         }
-        vitalsForm.addEventListener('submit', vitalsFormSubmitted);
+        vitalsForm.addEventListener("submit", vitalsFormSubmitted);
```

Comment:
Can we remove the style changes, same as above.

Reply:
Removed the styling/formatting changes.